vuecomponents/tool/src/main/frontend/tests/unit/sakai-avatar.spec.js

```diff
@@ -0,0 +1,154 @@
+/*
+ *  https://cli.vuejs.org/core-plugins/unit-jest.html
+ *  https://vue-test-utils.vuejs.org/
+ */
+
+import { shallowMount } from "@vue/test-utils";
```

Comment:
I was doing karma testing on the lit stuff, but it caused problems with the build servers, ate a load of resources. Earle asked me to remove them and that's why I don't have any UI tests on the lit components. This may happen with the vue tests, too.

Reply:
We could add add a build variable like -Dsakai.skip.test.node to skip that kind of tests. Then we can still run them locally. Or not let them run automatically. But I think it's worth keeping them. What do you think?

Reply:
Please PLEASE keep the unit tests. Even if they eat resources.

Reply:
Just leave them in for now and see what happens. I'm just warning you that's what happened with me.